nilearn/decoding/tests/test_decoder.py

```diff
@@ -143,7 +144,7 @@ def test_parallel_fit():
                                           is_classification=False,
                                           scorer=scorer,
                                           mask_img=None, class_index=1,
-                                          screening_percentile=None)))
+                                          screening_percentile=None, do_clustering=False, clustering_percentile=100)))
```

Comment:
I realize that ' do_clustering=False, clustering_percentile=100' are redundant. Can we get rid of `do_clustering` ?